src/System.Transactions.Local/tests/AsyncTransactionScopeTests.cs

```diff
@@ -1115,30 +1124,40 @@ public void VerifyBYOTOpenConnSimulationTest()
         }
 
         [Fact]
-        public void VerifyBYOTSyncTSNestedAsync()
+        public async Task VerifyBYOTSyncTSNestedAsync()
         {
             string txId1;
             string txId2;
 
-            AssertTransactionNull();
+            await Task.Run(delegate
+            {
+                var tcs = new TaskCompletionSource<bool>();
+                var ignored = Task.Delay(1).ContinueWith(_ =>
+                {
+                    AssertTransactionNull();
 
-            CommittableTransaction tx = new CommittableTransaction();
-            Transaction.Current = tx;
+                    CommittableTransaction tx = new CommittableTransaction();
+                    Transaction.Current = tx;
 
-            txId1 = AssertAndGetCurrentTransactionId();
-            SyncTSL3AsyncTSL2NestedTxWork(false, false, false, true, false, false, txId1);
-            txId2 = AssertAndGetCurrentTransactionId();
+                    txId1 = AssertAndGetCurrentTransactionId();
+                    SyncTSL3AsyncTSL2NestedTxWork(false, false, false, true, false, false, txId1);
 
-            Transaction.Current = null;
-            tx.Commit();
+                    txId2 = AssertAndGetCurrentTransactionId();
 
-            Assert.Equal(txId1, txId2);
+                    Transaction.Current = null;
+                    tx.Commit();
 
-            AssertTransactionNull();
+                    Assert.Equal(txId1, txId2);
+
+                    AssertTransactionNull();
+
+                });
+                tcs.SetResult(true);
```

Comment:
Same question for all of these... I'm not clear what this is solving.  And in this case, any asserts that fail inside of the ContinueWith are just going to be ignored, since you're not looking at the ignored task anywhere.

Reply:
I have made changes to remove the extra Task.Delay and ContinueWith. @stephentoub please have another look.